kubectl_client.cr

```diff
@@ -1311,7 +1311,7 @@ module KubectlClient
       Log.debug { "resource_labels kind: #{kind} resource_name: #{resource_name}" }
       if kind.downcase == "service"
         resp = resource(kind, resource_name, namespace: namespace).dig?("spec", "selector")
-      elsif kind.downcase == "deployment" 
+      elsif kind.downcase == "deployment" || kind.downcase == "statefulset" || kind.downcase == "replicaset" || kind.downcase == "daemonset" || kind.downcase == "job"
```

Comment:
I'm not a crystal lang expert but shouldn't be better to use an `includes` function?

```suggestion
      elsif "deployment statefulset replicaset daemonset job".includes?(kind.downcase)
```

Reply:
Thanks for the optimization idea. I thought about that and at the end I found most optimal and readable to use case command instead of if. PR updated.